package.json

```diff
@@ -15,8 +13,12 @@
     "url": "https://github.com/readmeio/api.git"
   },
   "engines": {
-    "node": ">=10"
+    "node": "^12 || ^14",
+    "npm": "^7"
```

Comment:
@todo remove this because it might prevent people on npm@6 from installing the packages